### PR TITLE
[ADF-5532] Adding upgrade411-50.md file

### PR DESCRIPTION
**Pull request by rbahirsheth**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/ADF-5532

**What is the new behaviour?**
Added ADF v4.11-v5.0.md file.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: docs/upgrade-guide/upgrade411-50.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+Title: Upgrading from ADF v4.11 to v5.0
+---
+
+# Upgrading from ADF v4.11 to v5.0
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:5]: what about breaking changes and how to upgrade existing apps?

File: docs/upgrade-guide/upgrade411-50.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+**Note:** the steps described below might involve making changes
+to your code. If you are working with a versioning system then you should
+commit any changes you are currently working on. If you aren't using versioning
+then be sure to make a backup copy of your project before going ahead with the
+upgrade.
+
+## Library updates
+
+### Automatic update using the Yeoman Generator
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:20]: yeoman generator does not perform any updates of the applications, it only generates new project scaffolds You don't need documenting how to create a yeoman app for people that want to upgrade the ADF

Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md]: Okay. Removed it.

File: docs/upgrade-guide/upgrade411-50.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+then be sure to make a backup copy of your project before going ahead with the
+upgrade.
+
+
+### Manual update
+
+1.  Update the `package.json` file with the latest library versions:
+    ```json
+    "dependencies": {
+        ...
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:24]: I suggest removing non-json elements, as the document is marked with red as incorrect.. no need putting "..." around the libraries

File: docs/upgrade-guide/upgrade411-50.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    To migrate custom code and application to Angular 14, please refer to the [offcial documentation](https://angular.io/).
+
+    **Note:**: Consider the possibility of leveraging [ADF v5.0.0-angular.13.2](https://www.npmjs.com/package/@alfresco/adf-core/v/5.0.0-angular.13.2), a       version of ADF compatible with Angular 13 that is meant to be used as intermediate step towards ADF v5. with angular v14.
+    
+4.  A breaking change worth mentioning is related to style import. with ADF v5.0.0 an extra ```\lib``` is required.
+
+
+    **example**:
+
+    previous versions of ADF :  ```@import '~@alfresco/adf-core/prebuilt-themes/adf-blue-orange.css';```
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:65]: for inline blocks, can be just single tick
```suggestion
    previous versions of ADF :  `@import '~@alfresco/adf-core/prebuilt-themes/adf-blue-orange.css';`
```

File: docs/upgrade-guide/upgrade411-50.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    
+    starting ADF 5.0.0 : ```@import '~@alfresco/adf-core/lib/prebuilt-themes/adf-blue-orange.css';```
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:67]:
```suggestion
    starting ADF 5.0.0 : `@import '~@alfresco/adf-core/lib/prebuilt-themes/adf-blue-orange.css';`
```

File: docs/upgrade-guide/upgrade411-50.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    | v13 |  [Changes & Deprecations](https://v13.angular.io/guide/update-to-latest-version)|
+    | v14 | [Changes & Deprecations](https://angular.io/guide/update-to-latest-version) |
+
+    ## Upgrade
+
+    To migrate custom code and application to Angular 14, please refer to the [offcial documentation](https://angular.io/).
+
+    **Note:**: Consider the possibility of leveraging [ADF v5.0.0-angular.13.2](https://www.npmjs.com/package/@alfresco/adf-core/v/5.0.0-angular.13.2), a       version of ADF compatible with Angular 13 that is meant to be used as intermediate step towards ADF v5. with angular v14.
+    
+4.  A breaking change worth mentioning is related to style import. with ADF v5.0.0 an extra ```\lib``` is required.
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:60]:
```suggestion
4.  A breaking change worth mentioning is related to style import. with ADF v5.0.0 an extra `\lib` is required.
```

File: docs/upgrade-guide/upgrade411-50.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    | v11 | [Changes & Deprecations](https://v11.angular.io/guide/updating-to-version-11)|
+    | v12 | [Changes & Deprecations](https://v12.angular.io/guide/updating-to-version-12)|
+    | v13 |  [Changes & Deprecations](https://v13.angular.io/guide/update-to-latest-version)|
+    | v14 | [Changes & Deprecations](https://angular.io/guide/update-to-latest-version) |
+
+    ## Upgrade
+
+    To migrate custom code and application to Angular 14, please refer to the [offcial documentation](https://angular.io/).
+
+    **Note:**: Consider the possibility of leveraging [ADF v5.0.0-angular.13.2](https://www.npmjs.com/package/@alfresco/adf-core/v/5.0.0-angular.13.2), a       version of ADF compatible with Angular 13 that is meant to be used as intermediate step towards ADF v5. with angular v14.
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:58]: extra spaces in the "a version"

File: docs/upgrade-guide/upgrade411-50.md (hunk 3 above)
Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md:24]: Updated.

File: docs/upgrade-guide/upgrade411-50.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    | Angular | 14 |
+    | Typescript | 4.6 |
+
+    For a complete list of changes, supported browsers and new feature please refer to the official documentation
+
+    | Angular version | link |
+    | --- | --- |
+    | v11 | [Changes & Deprecations](https://v11.angular.io/guide/updating-to-version-11)|
+    | v12 | [Changes & Deprecations](https://v12.angular.io/guide/updating-to-version-12)|
+    | v13 |  [Changes & Deprecations](https://v13.angular.io/guide/update-to-latest-version)|
```
[the remark this answers is not in the document]
Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md]: Removed extra space.

File: docs/upgrade-guide/upgrade411-50.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+
+5.  Reinstall your dependencies
+    ```sh
+    npm install
+    ```
+    
+6.  Run the application
+    ```sh
+    npm run start
```
[the remark this answers is not in the document]
Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md]: Updated.

File: docs/upgrade-guide/upgrade411-50.md (hunk 4 above)
Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md]: Updated.

File: docs/upgrade-guide/upgrade411-50.md (hunk 5 above)
Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md]: Updated.

File: docs/upgrade-guide/upgrade411-50.md (hunk 6 above)
Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md]: Updated.

File: docs/upgrade-guide/upgrade411-50.md (hunk 7 above)
Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md]: Removed extra space.

File: docs/upgrade-guide/upgrade411-50.md (hunk 3 above)
Comment (DenysVuika) [reply docs/upgrade-guide/upgrade411-50.md:24]: it is still there

File: docs/upgrade-guide/upgrade411-50.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+to your code. If you are working with a versioning system then you should
+commit any changes you are currently working on. If you aren't using versioning
+then be sure to make a backup copy of your project before going ahead with the
+upgrade.
+
+
+### Manual update
+
+1.  Update the `package.json` file with the latest library versions:
+    ```
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:22]:
```suggestion
    ```json
```

File: docs/upgrade-guide/upgrade411-50.md (hunk 3 above)
Comment (DenysVuika) [reply docs/upgrade-guide/upgrade411-50.md:24]: it is still there... non-json elements means everything that is not json, in your case it is tree dots as I've said before

File: docs/upgrade-guide/upgrade411-50.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+    | v14 | [Changes & Deprecations](https://angular.io/guide/update-to-latest-version) |
+
+    ## Upgrade
+
+    To migrate custom code and application to Angular 14, please refer to the [offcial documentation](https://angular.io/).
+
+    **Note:**: Consider the possibility of leveraging [ADF v5.0.0-angular.13.2](https://www.npmjs.com/package/@alfresco/adf-core/v/5.0.0-angular.13.2),
+    a version of ADF compatible with Angular 13 that is meant to be used as intermediate step towards ADF v5. with angular v14.
+    
+4.  A breaking change worth mentioning is related to style import. with ADF v5.0.0 an extra `\lib` is required.
```
Comment (DenysVuika) [inline docs/upgrade-guide/upgrade411-50.md:61]:
```suggestion
4.  A breaking change worth mentioning is related to style import. with ADF v5.0.0 an extra `/lib` is required.
```

File: docs/upgrade-guide/upgrade411-50.md (hunk 3 above)
Comment (rbahirsheth) [reply docs/upgrade-guide/upgrade411-50.md]: Done.

**Review by DenysVuika — APPROVED** [review]

**Comment by rbahirsheth** [comment]
Closing this PR cause I have addressed this issue & raised a new PR [8516](https://github.com/Alfresco/alfresco-ng2-components/pull/8516)

**Pull request closed without merging**